### PR TITLE
Add contribution guidance for Tools page

### LINKED ISSUE #134: Update contribution guidance for adding new tools to the list
There have been a few requests to maintain a list of packages that are useful for government analysis. We should update the Tools page to indicate when packages have been developed by government, to celebrate these.

We should update the contribution guidance or Tools page to advise on how new tools can be added to the list by Pull Request or by raising a new Issue.

### PR DESCRIPTION
**Pull request by foster999**
Aiming to get more user input on tools that are useful for reproducible analysis. Adding clear guidance on how to contribute to this page.

### CODE REVIEW

File: book/tools.md [hunk 1]
```diff
@@ -1,67 +1,96 @@
 # Tools
 
-This section points you to useful tools that support reproducible analysis.
+This section highlights tools that support reproducible analysis and research.
+This includes tools for general software development and bespoke packages that have been developed for government analysis.
+Those developed or contributed to within government are marked with an asterisk (*).
 
+If you have developed a package for use in analysis or recommend any that are not included here, please add them to the list.
+You can request a new tool to be added to the list by [creating an issue on GitHub](https://github.com/best-practice-and-impact/qa-of-code-guidance/issues/new/choose)
+or [contacting us by email](mailto:ASAP@ons.gov.uk?subject=Duck%20Book%20Tools)
```
Comment (rowanhemsi) [inline book/tools.md]: Missing full stop after link

File: book/tools.md [hunk 2]
```diff
@@ -1,67 +1,96 @@
...
+Alternatively, you can add it directly to the project by creating a Pull Request.
+You can do this using the "Suggest edit" link under the GitHub logo at the top of this page.
+Please include a link and brief description when requesting a new tool to be added.
```
Comment (rowanhemsi) [inline book/tools.md:12]: Do we want an issue template for tool requests?

File: book/tools.md [hunk 3]
```diff
@@ -1,67 +1,96 @@
...
-* [patrick](https://github.com/google/patrick)
-* [rhub](https://r-hub.github.io/rhub/)
+* [dplyr](https://dplyr.tidyverse.org/)  - common data analysis and manipulation
+* [sparklyr](https://spark.rstudio.com/) - for distributed (large) data
 
+## Publishing
 
-## Test Coverage
+* [Quarto](https://quarto.org/) - reproducible documents for Python and R
+* [a11tables (R only)](https://co-analysis.github.io/a11ytables/index.html) * - creating reproducible, accessible spreadsheets
```
Comment (rowanhemsi) [inline book/tools.md]: Missing "y" in a11ytables

File: book/tools.md [hunk 4]
```diff
@@ -1,67 +1,96 @@
...
+## Version control
 
+* [Git](https://git-scm.com/) - common open source version control system
+* [pre-commit](https://pre-commit.com/) - trigger checks (e.g. linters and formatters) before Git commits are created
 
-### Python
+## Project templates
+
+* [govcookiecutter (Python)](https://github.com/best-practice-and-impact/govcookiecutter) * - template project for reproducible analysis
+* [Rgovcookiecutter (R)](https://github.com/best-practice-and-impact/Rgovcookiecutter) * - template project for reproducible analysis
```
Comment (rowanhemsi) [inline book/tools.md]: Does Rgovcookiecutter exist yet?

File: book/tools.md [hunk 5]
```diff
@@ -76,48 +105,24 @@ Automated code format repair.
...
-
 ## Packaging Code
 
+Creating and releasing code as a package.
 
 ### Python
 
-* [twine](https://pypi.org/project/twine/)
-
+* [twine](https://pypi.org/project/twine/) - utility for publishing Python packages to [the Python Package Index PyPI](https://pypi.org/)
```
Comment (rowanhemsi) [inline book/tools.md:119]: Could add poetry here or in dependency management

File: book/tools.md [hunk 6]
```diff
@@ -128,28 +133,3 @@ Automated code format repair.
 * [Travis](https://travis-ci.org/)
```
Comment (rowanhemsi) [inline book/tools.md:138]: Is Travis still recommended?

File: README.md [hunk 7]
```diff
@@ -9,17 +9,21 @@ This document forms part of the Quality Guidance, published by the Quality and I
...
 ### Getting started
 
-To start contributing, you'll need python installed.
+Minor text edits can be submitted as a Pull Request using the "Suggest edit" button under the GitHub logo at the top of the page you would like to change.
+
+For changes to anything other than lines of text, you should follow these steps to make the changes locally:
+
+To start contributing, you'll need Python installed.
 If you sit outside of Quality and Improvement Division, the you'll need to [create a Fork of this repository to make changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks).
 Once forked, you should clone the fork repository to get a copy of the book. Then install it's Python dependencies like so:
```
Comment (rowanhemsi) [inline README.md]: I think this should be its not it's

**Review by rowanhemsi — CHANGES_REQUESTED** [review]
A couple of comments but in general this looks like a good update

File: book/tools.md (hunk 2 above)
Comment (rowanhemsi) [reply book/tools.md:12]: Issue added #155

File: book/tools.md [hunk 8]
```diff
@@ -1,67 +1,96 @@
 # Tools
 
-This section points you to useful tools that support reproducible analysis.
+This section highlights tools that support reproducible analysis and research.
+This includes tools for general software development and bespoke packages that have been developed for government analysis.
+Those developed or contributed to within government are marked with an asterisk (*).
```
Comment (rowanhemsi) [inline book/tools.md]: Not all screen readers will read out `*`, so this should be changed for accessibility reasons

**Comment by rowanhemsi** [comment]
Do we need to add a disclaimer saying that whilst we believe these tools are useful, we have not comprehensively quality assured or security checked them all and are not liable should any issues arise

File: book/tools.md (hunk 5 above)
Comment (rowanhemsi) [reply book/tools.md:119]: Added to issue #156

File: book/tools.md (hunk 6 above)
Comment (rowanhemsi) [reply book/tools.md:138]: Added to issue #156

File: book/tools.md (hunk 4 above)
Comment (rowanhemsi) [reply book/tools.md]: Added to issue #156

File: book/tools.md (hunk 8 above)
Comment (rowanhemsi) [reply book/tools.md]: Asterisk `*` replaced with `(gov)`, which should be more accessible and explicit. Feel free to change if you have an alternative suggestions

**Review by robertswh — APPROVED** [review]
Reviewed disclaimer wording on using the tools.

**Review by rowanhemsi — APPROVED** [review]
Changes requested have been implemented or an issue created to discuss. Happy for this to now be merged

**Pull request merged**